### ISSUE TITLE
dxf2png and image does not close 

### ISSUE THREAD

**Issue by yingma111** [mozman/ezdxf#224]
Hello, I am working on converting a DXF file to PNG format, using one of the add-ons tool in ezdxf.
Below is the script that I ran to output an image file.

```
import matplotlib.pyplot as plt
import ezdxf
from ezdxf.addons.drawing import RenderContext, Frontend
from ezdxf.addons.drawing.matplotlib_backend import MatplotlibBackend

def dxf2png(path_to_dxf='', path_to_output_png=''):
    doc = ezdxf.readfile(path_to_dxf)
    layout = doc.modelspace()
    fig = plt.figure()
    ax = fig.add_axes([0, 0, 1, 1])
    ctx = RenderContext(doc)
    out = MatplotlibBackend(ax)
    Frontend(ctx, out).draw_layout(layout, finalize=True)
    fig.savefig(path_to_output_png, dpi=300)
    plt.close(fig)


if __name__ == "__main__":
    dxf_file = "../../DXFFILE/batch3/E/(29).dxf"
    s = dxf_file[dxf_file.rfind("/") + 1:dxf_file.find(")") + 1]
    png_file = "images_ezdxf/E/" + s + ".png"
    dxf2png(path_to_dxf=dxf_file, path_to_output_png=png_file)

```
Output is below:
![(29)](https://user-images.githubusercontent.com/53547240/92346217-a6432880-f0fe-11ea-871a-0daa2bb9279d.png)

But, I want result is:
![捕获](https://user-images.githubusercontent.com/53547240/92346327-f91ce000-f0fe-11ea-8fba-7fcdfcc79f82.JPG)

And dxf file is:

[(29).dxf.tar.gz](https://github.com/mozman/ezdxf/files/5181052/29.dxf.tar.gz)

**Comment by mozman** [comment]
@mbway first test shows a correct ELLIPSE calculation and a correct Path() approximation:

![path_rendering](https://user-images.githubusercontent.com/4230966/92351732-adf1d580-f0dc-11ea-987d-9668cbd1a344.png)

**Comment by mozman** [comment]
@yingma111 this is fixed in the latest version v0.14 which will be released soon, but you can install a beta version:
```
pip install ezdxf==0.14b3
```
An import has to be modified:
```Python
...
from ezdxf.addons.drawing.matplotlib import MatplotlibBackend
...
```


![224](https://user-images.githubusercontent.com/4230966/92352217-de863f00-f0dd-11ea-80b0-672e20fdff3f.png)

**Comment by yingma111** [comment]
Super thanks!
